uk.co.compucorp.civicrm.hrleaveandabsences/xml/option_groups/leave_request_status_install.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="iso-8859-1" ?>
+
+<CustomData>
+
+  <OptionGroups>
+    <OptionGroup>
+      <name>hrleaveandabsences_leave_request_status</name>
+      <title>Leave Request Status</title>
+      <is_reserved>1</is_reserved>
+      <is_active>1</is_active>
+    </OptionGroup>
+  </OptionGroups>
+
+  <OptionValues>
+    <OptionValue>
+      <label>Approved</label>
+      <value>1</value>
+      <name>approved</name>
+      <is_default>0</is_default>
+      <weight>0</weight>
```

Comment:
all the option values has the same weight here which is 0 , is this intentional or just a mistake ?